src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -7,9 +7,12 @@ public class CliSyntax {
 
     /* Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
+    public static final Prefix PREFIX_CLASS_GROUP = new Prefix("c/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
+    public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
```

Comment:
Are we able to delete all references to PREFIX_ADDRESS since we no longer store address as a field? Maybe we can do this in another pull request later.

Reply:
Yep might have to do this in another PR - the prefix_address is used in many other commands so I don't want to mess that up in this PR